__tests__/app.test.js

```diff
@@ -99,6 +98,49 @@ describe("Backend testing", () => {
         });
     });
   });
-=======
->>>>>>> main
+  describe.only("PATCH /api/articles/:article_id", () => {
+    test("status: 200, responds with the updated article", () => {
+      const id = 1;
+      const voteIncrement = { votes: -4 };
+      return request(app)
+        .patch(`/api/articles/${id}`)
+        .send(voteIncrement)
+        .expect(200)
+        .then(({body}) => {
+          // console.log(body)
+          expect(body.article).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 96,
+          });
+          expect(body.article.votes).toBe(96);
+        });
+    });
+    test("status: 400, bad request invalid data", () => {
+      const id = 1;
+      const voteIncrement = { votes: "nothing to see here" };
+      return request(app)
+        .patch(`/api/articles/${id}`)
+        .send(voteIncrement)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid id/vote");
+        });
+    });
+    test("status: 400, bad request when passed an empty object increment ", () => {
+      const id = 1;
+      const voteIncrement = {};
+      return request(app)
+        .patch(`/api/articles/${id}`)
+        .send(voteIncrement)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("vote increment has not been provided");
+        });
+    });
+  });
```

Comment:
What happens if the id is not valid- `/api/articles/apples`? I would add a test for this to make sure it has been handled.